packages/graphql-mini-transforms/package.json

```diff
@@ -38,17 +38,17 @@
   },
   "dependencies": {
     "@jest/transform": "^27.0.2",
-    "@types/fs-extra": "^9.0.11",
-    "@types/webpack": "^4.41.11",
     "fs-extra": "^9.1.0",
     "graphql": ">=14.5.0 <16.0.0",
-    "graphql-typed": "^1.1.1",
-    "loader-utils": "^2.0.0"
+    "graphql-typed": "^1.1.1"
   },
   "devDependencies": {
     "@types/common-tags": "^1.8.0",
-    "@types/loader-utils": "^1.1.3",
-    "common-tags": "^1.8.0"
+    "common-tags": "^1.8.0",
+    "webpack": "^5.40.0"
+  },
+  "peerDependencies": {
+    "webpack": ">=5.38.0"
```

Comment:
Is webpack here used just for types? If that's the case can we mark it as an optional peer dependency? https://github.com/Shopify/loom/blob/06c44051385af276a1982a583b34aeac6160c75f/packages/loom-plugin-babel/package.json#L45-L48

Reply:
Good catch. Updated.